src/App.js

```diff
@@ -1,7 +1,30 @@
 import React from "react";
+import DesktopNavigationBar from "../src/Components/UI/Navigation/DesktopNavigationBar";
+import LandingPage from "./Pages/Landing/LandingPage";
+import { useModal } from "./Context/ModalContext";
+import Login from "../src/Components/UI/Modal/Login";
+import Signup from "../src/Components/UI/Modal/Signup";
+import MobileNavigationBar from "../src/Components/UI/Navigation/MobileNavigationBar";
+import MobileNavigationBarBottom from "../src/Components/UI/Navigation/MobileNavigationBarBottom";
+import "./App.css";
+import Footer from "./Components/UI/Footer/Footer";
+import BodyWrapper from "./Components/UI/Wrapper/BodyWrapper";
 
 const App = () => {
-  return <h1>App</h1>;
+  const { showLoginModal, showSignupModal } = useModal();
+  return (
+    <>
+      {showLoginModal && <Login />}
+      {showSignupModal && <Signup />}
```

Comment:
emphasis on using more ternary operator than &&

Reply:
Ok, will do that.